examples/cmd/snapshotting/Makefile

```diff
@@ -18,8 +18,15 @@ ARCH=$(shell uname -m)
 GID = $(shell id -g)
 
 PWD=$(shell pwd)
+GOBIN=$(PWD)/bin
 FC_TEST_DATA_PATH?=$(PWD)
 
+define install_go
+	cd .hack \
+		GO111MODULE=on GOBIN=$(GOBIN) go get $(1) && \
+		GO111MODULE=on GOBIN=$(GOBIN) go install $(1)
```

Comment:
Do we need to do an `|| true` here since one will fail?

Reply:
No, it seems like what we actually need is this (ignoring setting env vars for clarity):

go 1.14/1.15:
```
go get $PACKAGE@$VERSION
go install $PACKAGE
```
go 1.16+
```
go install $PACKAGE@$VERSION
```

I've been banging my head against a wall trying to figure out a reasonable way to do this either in the Makefile or in portable shell scripting, but maybe it's worth just having the 10 or so gross if/else make rules.